code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -49,6 +49,14 @@
 	icon_state = "tile-wood"
 	turf_type = /turf/simulated/floor/wood
 
+/obj/item/stack/tile/wood/cyborg

```

Comment:
there's literally no reason for this to exist, and there is a technical hard-cap on paths